client/gutenberg/editor/calypsoify-iframe.jsx

```diff
@@ -64,16 +64,18 @@ class CalypsoifyIframe extends Component {
 	}
 
 	onMessage = ( { data, origin } ) => {
```

Comment:
to make variable assignment consistent… (referring to what @obenland brought up in person)

```js
onMessage = ( { data = { action, type } = {}, origin } ) => {
	if ( type !== 'gutenbergIFrameMessage' ) {
		return;
	}

	…

	if ( 'loaded' === action ) {
		…
	}

	…
}
```